technical_architecture/performances_guide/batch_page_size.rst

```diff
@@ -0,0 +1,13 @@
+Job product batch size
+======================
+
+A product catalog is unique to one's need. It is configured with a custom number of attributes, families, locales, channels, etc. Those different combinations may affect the import/exports performances.
+
+Indeed the size of the products will define how much RAM they will take to be processed. During product import/exports, we regularly flush the processed products into the database to avoid too much memory consumption.
+
+But, if we flush them too often, the transaction time can slow the process: flushing is not free and doing it too often will be inefficient. On the other hand, if we flush too sporadically, the RAM consumption can increase to a point where the PHP's garbage collector is called too many times and slows down the import/exports.
+
+That's why there is no universal answer to the `pim_job_product_batch_size` parameter and if you encounter performance issues on your import/exports, you may need to tweak it.
+
+To do so, try to increase it if your batch job seems unexpectedly slow and your product have a reasonable size. Or try to reduce it if the batch processes are too slow and your product have a lot of product values. To know if your products are above of below average size, refer to :the performance guide:`/maintain_pim/scalability_guide/representative_catalogs`.
```

Comment:
We should definitively delete these "representative catalogs" (part of my todo list ...). It's totally outdated.

Reply:
ok, what can I do in the meantime?

Reply:
I think we should not refer to the page.